src/js/windows/shot-generator/main.js

```diff
@@ -39,7 +39,10 @@ const show = (onComplete) => {
     show: false,
     center: true,
     frame: true,
-    titleBarStyle: 'default', //'hiddenInset',
+
```

Comment:
weird that github is showing this as a diff... this code already exists in `shot-generator`...